src/assets/styles/global.scss

```diff
@@ -1,22 +1,42 @@
 @import "./node_modules/@esri/calcite-colors/colors.scss";
 @import "./node_modules/@esri/calcite-base/dist/_index.scss";
-@import "../../scss/global";
+
+/* CSS vars (@include in global) */
+@import "layout";
+@import "shadow";
 
 :root {
   font-family: $avenir-family;
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  --calcite-border-radius: 3px;
+
   @include calcite-theme-light();
+  @include componentLayoutVars();
+  @include componentShadowVars();
+
+  --calcite-border-radius: 3px;
+  --calcite-ui-opacity-disabled: 0.3;
+
+  --calcite-ui-text-weight: 400;
```

Comment:
Maybe we have a typography partial inside calcite-components?

Reply:
Those should probably live in base, ideally IMO.

Reply:
I agree. Maybez we can do that after this PR?

Reply:
Also for some reason, these font weights only work sometimes. 
Like in FlowItem, PickListItem, ValueListItem, the font-weights mostly work. (But -light [300] doesn't seem to work but [200] does... 🤷‍♂️ )
But in Action and Block's header, the font-weights aren't working as intended.
This can be handled in a different issue unless anybody has a quick fix.

Reply:
font weights are declared in calcite-fonts as `300,400,500,700`, not 600.